dotcom-rendering/src/web/components/FrontPicture.tsx

```diff
@@ -0,0 +1,152 @@
+import { css } from '@emotion/react';
+import { breakpoints } from '@guardian/source-foundations';
+import React from 'react';
+import type { ImageWidthType } from './Picture';
+import { descendingByBreakpoint, generateImageURL } from './Picture';
+
+type Props = {
+	imageSize: ImageSizeType;
```

Comment:
This is required, which is sound 👍  but I notice we're [defaulting to small in Card](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/web/components/Card/Card.tsx#L194), which we might need to remove in a later refactor